src/main/java/edu/kis/powp/jobs2d/command/CommandBoundariesCheckVisitor.java

```diff
@@ -4,11 +4,11 @@
 
 import static java.lang.Math.abs;
 
-public class CommandBoundariesCheckVisitor implements Visitor{
-	private final ICanvas canvas;
+public class CommandBoundariesCheckVisitor implements Visitor, ICommandBoundariesCheckStrategy{
+	private final IRectangularCanvas canvas;
```

Comment:
CommandBoundariesCheckVisitor doesn't need to assume work with rectangular canvas. It is an unnecessary limitation.